console/console-init/ui/src/modules/project/containers/ProjectToolbarContainer/ProjectToolbarContainer.tsx

```diff
@@ -53,9 +59,25 @@ export const ProjectToolbarContainer: React.FunctionComponent<IProjectToolbarCon
     setFilter(initialiseFilterForProject());
   };
 
-  const onSelectDeleteAll = async (event: any) => {
-    if (event.target.value === "deleteAll") {
-      await onDeleteAll();
+  const onSelectKebabOption = async (event: any) => {
+    const option = event.target.value;
+    if (option) {
+      switch (option) {
+        case "deleteAll":
+          await onDeleteAll();
+          break;
+        case "enableAll":
+          await onToggleAll("enable");
+          break;
+        case "disableAll":
+          await onToggleAll("disable");
+          break;
+        default:
+          break;
+      }
+      if (option === "deleteAll") {
```

Comment:
why do you need this `if` code here. we have already covered onDeleteAll in switch ?